crates/nu-command/src/filters/split_by.rs

```diff
@@ -21,7 +21,7 @@ impl Command for SplitBy {
     }
 
     fn usage(&self) -> &str {
-        "Create a new table splitted."
+        "Create a new table split."
```

Comment:
I would leave this one as-is. 

"Create a new table split." doesn't make sense, and the `split-by` command is an unusual command that we will probably get rid of eventually.

Reply:
How about `Create a new table that has been split by the passed in column name`?